packages/deployer/test/fixture-projects/sample-project/test/UpgradeModule.test.js

```diff
@@ -55,7 +55,7 @@ describe('UpgradeModule', () => {
     });
   });
 
-  describe('when the owner attempts to upgrade to a sterile implementation', () => {
+  describe.skip('when the owner attempts to upgrade to a sterile implementation', () => {
```

Comment:
skipped since the anti-bricking protection is disabled. Will be set back (and moved to core-contracts) with issue 214

Reply:
What do you think about making the sample-project tests "dumber"? I.e. not check for the full functionality, but rather just check that it can upgrade.

IMO the full tests coverage should live in core-contracts and this should just serve as an example.

Reply:
is what I said in the comment. It will be moved to core-contracts together with the fix for issue 214